### PR TITLE
markdownlint lambdas and py-shared

### PR DESCRIPTION
**Pull request by sir-sigurd**
This PR fixes some markdownlint complaints and ignores others in the most specific way.
The main intention is to unignore files we'll likely modify in future e.g. changelogs.

### CODE REVIEW

File: lambdas/indexer/test/README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Rudimentary tests
 
 ## TODO
+
 * [ ] run inside of Docker container that mimics Lambda env
-		* https://blog.quiltdata.com/an-easier-way-to-build-lambda-deployment-packages-with-docker-instead-of-ec2-9050cd486ba8
-		* https://nvbn.github.io/2015/09/09/pytest-docker/
+  *<https://blog.quiltdata.com/an-easier-way-to-build-lambda-deployment-packages-with-docker-instead-of-ec2-9050cd486ba8>
+  * <https://nvbn.github.io/2015/09/09/pytest-docker/>
```
Comment (fiskus) [inline lambdas/indexer/test/README.md:7]: Huh?

**Review by fiskus — COMMENTED** [review]
Nice, but there is a mistyping in `lambdas/indexer/test/README.md`?

File: lambdas/indexer/test/README.md (hunk 1 above)
Comment (fiskus) [reply lambdas/indexer/test/README.md:7]: Should there be the nested list with links?

image

Comment (sir-sigurd) [reply lambdas/indexer/test/README.md:7]: fixed, thanks!

**Review by fiskus — APPROVED** [review]

**Pull request merged**